src/editor/utils/exportUnstylishHtml.js

```diff
@@ -1,7 +1,10 @@
+import createDOMPurify from 'dompurify'
 import marked from '../parser/marked'
+import { DOMPURIFY_CONFIG } from '../config'
 
 const exportUnstylishHtml = markdown => {
-  const html = marked(markdown)
+  const DOMPurify = createDOMPurify(window)
+  const html = DOMPurify.sanitize(marked(markdown), DOMPURIFY_CONFIG)
```

Comment:
Because DOMPurify will remove all the `script` tags, But I think we should leave it as escaped text.

So I think we need escape the html before use `sanitize`.

```javascript
const html = DOMPurify.sanitize(escapeInBlockHtml(marked(markdown)), DOMPURIFY_CONFIG)
```